Dart/testSrc/com/jetbrains/lang/dart/ide/editor/DartWordSelectionTest.java

```diff
@@ -0,0 +1,56 @@
+package com.jetbrains.lang.dart.ide.editor;
+
+import com.intellij.testFramework.fixtures.CodeInsightTestUtil;
+import com.jetbrains.lang.dart.DartCodeInsightFixtureTestCase;
+import com.jetbrains.lang.dart.util.DartTestUtils;
+
+public class DartWordSelectionTest extends DartCodeInsightFixtureTestCase {
+
+  @Override
+  public void setUp() throws Exception {
+    super.setUp();
+    DartTestUtils.configureDartSdk(myModule, getTestRootDisposable(), true);
```

Comment:
I'll change the last argument to `false`, we do not need real Dart SDK and real Analysis Server for these tests